src/screens/TopicPageScreen/index.js

```diff
@@ -424,6 +433,9 @@ const TopicPageScreen = (props) => {
         setIsFollow={setIsFollow}
         isFollow={isFollow}
         getTopicDetail={getTopicDetail}
+        onFollowButtonPress={() => bottomSheetFollowRef?.current?.open()}
+        followType={followType}
+        setFollowType={setFollowType}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onFollowButtonPress` function has been updated to open the `bottomSheetFollowRef`. The `followType` and `setFollowType` props have also been added. Ensure that these changes are reflected in the component's PropTypes.
```diff
+ onFollowButtonPress={() => bottomSheetFollowRef?.current?.open()}
+ followType={followType}
+ setFollowType={setFollowType}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->